onnx/defs/quantization/defs.cc

```diff
@@ -101,4 +102,49 @@ ONNX_OPERATOR_SET_SCHEMA(
               updateOutputShape(ctx, 0, input_shape);
             }));
 
+static const char* FusedQuantizeLinear_ver11_doc = R"DOC(
+A Function to fuse calculation for Scale, Zero Point and FP32->8Bit convertion of FP32 Input data.
+Outputs Scale, ZeroPoint and Quantized Input for a given FP32 Input)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    FusedQuantizeLinear,
+    11,
+    OpSchema()
+        .SetDoc(FusedQuantizeLinear_ver11_doc)
+        .Input(0, "X", "Input tensor", "T1")
+        .Output(0, "Y", "Quantized output tensor", "T2")
+        .Output(1, "Y_Scale", "Output Scale. It's a scalar or a 1D tensor with size 1.", "tensor(float)")
+        .Output(2, "Y_Zero_Point", "Output Zero point. It's a scalar or a 1D tensor of size 1.", "T2")
+        .Attr(
+          "to",
+          "The data type to which the elements of the input tensor are quantized to. Strictly must be one of the types from DataType enum in TensorProto",
```

Comment:
I assume that this is not being used currently and that T2 is restricted to one type (until Functions are extended to simplify coding this up)? It may be worth mentioning this in the documentation.

Reply:
Yes, I will add it in the doc